zerver/tests/test_new_users.py

```diff
@@ -96,6 +96,7 @@ def test_get_os_on_new_login(self):
 
 
 class TestNotifyNewUser(ZulipTestCase):
+
```

Comment:
sorry I'm just now noticing your comments below. Removing extra line as requested and thanks for the heads up!